### PR TITLE
Proposal to support notation when formatting currencies

### PR DESCRIPTION
**Pull request by jacoblukas**

### CODE REVIEW

File: Proposals/NNNN-notation-formatting-for-currencies.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Support notation when formatting currencies
+
+* Proposal: [SF-NNNN](NNNN-notation-formatting-for-currencies.md)
+* Authors: [Jacob Lukas](https://github.com/jacoblukas)
+* Review Manager: TBD
```
Comment (itingliu) [inline Proposals/NNNN-notation-formatting-for-currencies.md]:
```suggestion
* Review Manager: [Tina Liu](https://github.com/itingliu)
```

File: Proposals/NNNN-notation-formatting-for-currencies.md
Comment (itingliu) [inline Proposals/NNNN-notation-formatting-for-currencies.md:1]: Please use 0008

File: Proposals/NNNN-notation-formatting-for-currencies.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# Support notation when formatting currencies
+
+* Proposal: [SF-NNNN](NNNN-notation-formatting-for-currencies.md)
```
Comment (itingliu) [inline Proposals/NNNN-notation-formatting-for-currencies.md]:
```suggestion
* Proposal: [SF-0008](0008-notation-formatting-for-currencies.md)
```

File: Proposals/NNNN-notation-formatting-for-currencies.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+* Authors: [Jacob Lukas](https://github.com/jacoblukas)
+* Review Manager: TBD
+* Status: **Draft**
```
Comment (itingliu) [inline Proposals/NNNN-notation-formatting-for-currencies.md]:
```suggestion
* Status: **Active review: Mar 04, 2024...Mar 11, 2024**
```

File: Proposals/NNNN-notation-formatting-for-currencies.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+* Bug: [apple/swift-foundation#379](https://github.com/apple/swift-foundation/issues/379)
+* Implementation: [apple/swift-foundation#NNNNN](https://github.com/apple/swift-foundation/pull/NNNNN)
```
Comment (itingliu) [inline Proposals/NNNN-notation-formatting-for-currencies.md]: Please update this when it's ready

**Review by parkera — APPROVED** [review]

**Review by itingliu — APPROVED** [review]

**Pull request merged**